packages/react-i18n/src/i18n.ts

```diff
@@ -237,22 +314,35 @@ function isTranslateOptions(
   return 'scope' in object;
 }
 
-function isSameMonthAndYear(source: Date, target: Date) {
-  return (
-    source.getFullYear() === target.getFullYear() &&
-    source.getMonth() === target.getMonth()
-  );
+enum TimeUnit {
```

Comment:
I plan to find a proper a home for this enum and the `isLessThan*` functions below. `@shopify/javascript-utilities/dates` perhaps?

Reply:
yea that probably makes sense